### PR TITLE
LPS-152124 Vocabularies sidebar adjustments

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/site-navigation/site-navigation-menu-item-asset-vocabulary/src/main/resources/META-INF/resources/js/AssetVocabularyContextualSidebar.js [hunk 1]
```diff
@@ -52,13 +52,35 @@ function AssetVocabularyContextualSidebar({
...
+				setCustomNameInvalid(true);
+			}
+		},
+		[customName]
+	);
+
+	useEffect(() => {
+		const submitButton = document.querySelector('button[type=submit]');
+
+		submitButton?.addEventListener('click', onFormSubmit);
```
Comment (p2kmgcl) [inline modules/apps/site-navigation/site-navigation-menu-item-asset-vocabulary/src/main/resources/META-INF/resources/js/AssetVocabularyContextualSidebar.js:77]: Do you think we can move the onFormSubmit function inside this hook? It is not being used anywhere else :shrug:

**Review by p2kmgcl — COMMENTED** [review]
Maybe a tiny refactor (I left a comment), but looks very nice :smile: 

I love the commit separation and messages, it is so easy to read :heart:

File: modules/apps/site-navigation/site-navigation-menu-item-asset-vocabulary/src/main/resources/META-INF/resources/js/AssetVocabularyContextualSidebar.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/site-navigation/site-navigation-menu-item-asset-vocabulary/src/main/resources/META-INF/resources/js/AssetVocabularyContextualSidebar.js:77]: Um, you are right, sure 🥰  that way we avoid the useCallback

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**